dashboard/app/models/peer_review.rb

```diff
@@ -0,0 +1,74 @@
+# == Schema Information
+#
+# Table name: peer_reviews
+#
+#  id              :integer          not null, primary key
+#  submitter_id    :integer
+#  reviewer_id     :integer
+#  from_instructor :boolean          default(FALSE), not null
+#  script_id       :integer          not null
+#  level_id        :integer          not null
+#  level_source_id :integer          not null
+#  data            :text(65535)
+#  status          :integer
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#
+# Indexes
+#
+#  index_peer_reviews_on_level_id         (level_id)
+#  index_peer_reviews_on_level_source_id  (level_source_id)
+#  index_peer_reviews_on_reviewer_id      (reviewer_id)
+#  index_peer_reviews_on_script_id        (script_id)
+#  index_peer_reviews_on_submitter_id     (submitter_id)
+#
+
+class PeerReview < ActiveRecord::Base
+  belongs_to :submitter, class_name: 'User'
+  belongs_to :reviewer, class_name: 'User'
+  belongs_to :script
+  belongs_to :level
+  belongs_to :level_source
+
+  REVIEWS_PER_SUBMISSION = 2
+
+  VALID_STATUSES = [
+    ACCEPTED = 0,
+    REJECTED = 1,
+    ESCALATED = 2
+  ]
+
+  def named_status
+    case status
+    when 0
+      'Accepted'
```

Comment:
Good point, we'll need to internationalize if this is used in student-facing scripts.  @laurelfan what do you think about calling `I18n.t` in models?  [We do this in a few other models](https://github.com/code-dot-org/code-dot-org/blob/63d195fcf24f6a911db0417aa557fdb05838899c/dashboard/app/models/stage.rb#L47-L53), but I'm not sure if this is an anti-pattern.